config/v1/types_cluster_version.go

```diff
@@ -215,6 +222,58 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionCapabilitiesDefault is a constant representing the
+// default policy for optional, core cluster components.
+type ClusterVersionCapabilitiesDefault string
+
+const (
+	// IncludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be included by default.
+	IncludeClusterVersionCapabilities = "Include"
+
+	// ExcludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be excluded by default.
+	ExcludeClusterVersionCapabilities = "Exclude"
+)
+
+// ClusterVersionCapabilitiesSpec configures the installation of
+// optional, core cluster components.
+// +k8s:deepcopy-gen=true
+type ClusterVersionCapabilitiesSpec struct {
+	// inclusionDefault declares the default policy for optional,
+	// core cluster components.
+
+	// +optional
+	InclusionDefault ClusterVersionCapabilitiesDefault `json:"inclusionDefault,omitempty"`
```

Comment:
What is the default inclusion default?

Reply:
Discussion of the set default value has moved to [this newer thread][1]; marking this one resolved.

[1]: https://github.com/openshift/api/pull/1106#discussion_r799817808